autoload/sy.vim

```diff
@@ -24,9 +24,18 @@ function! sy#start() abort
     return
   endif
 
+
+  function! s:chdir()
+    if has('nvim')
+      return haslocaldir(-1, 0) ? 'tcd' : haslocaldir() ? 'lcd' : 'cd'
```

Comment:
Wouldn't it have to be..

```vim
return haslocaldir() ? 'lcd' : (haslocaldir(-1, 0) ? 'tcd' : 'cd')
```

From narrowest (window) to broadest (global) scope? 

(Without warrant, it's 1:30 AM.)